docs/internals/release-process.txt

```diff
@@ -71,6 +53,54 @@ security purposes, please see :doc:`our security policies <security>`.
     So the answer to "should I upgrade to the latest minor release?" will always
     be "yes."
 
+Starting with Django 3.0, version numbers will use a loose form of `semantic
+versioning <http://semver.org/>`_. SemVer makes it easier to see at a glance
+how compatible releases are with each other. It also helps to anticipate when
+compatibility shims will be removed. It’s not a pure form of SemVer as each
+feature release will continue to have some documented backwards
+incompatibilities where a deprecation path isn't possible or not worth the
+cost. Also, deprecations from the last minor release in a series will be dropped
+in a non-dot-zero release to accommodate our policy of keeping deprecation
+shims for at least two releases. Read on to the next section for an example.
+
+.. _internal-release-deprecation-policy:
```

Comment:
why starting from Django 3.0 and not from 2.0 or 2.5 or whatever other random number version, you know you're planning about 8/10 years ahead of it.